src/helm/benchmark/run_specs.py

```diff
@@ -1793,6 +1813,40 @@ def get_wmt_14_spec(language_pair: str, max_train_instances: int = 1) -> RunSpec
     )
 
 
+def get_opinions_qa_spec(
+    survey_type: str,
+    num_logprobs: str,
+    context: str = "None",
+    num_train_trials: str = "1",
```

Comment:
I think you can get rid of this, because we have a RunExpander that allows you to set `num_train_trials`